utils/chain-ops/genesis/layer_one.go

```diff
@@ -132,7 +132,7 @@ func BuildL1DeveloperGenesis(config *DeployConfig) (*core.Genesis, error) {
 
 	data, err = sysCfgABI.Pack(
 		"initialize",
-		config.FinalSystemOwner,
+		config.SecurityCouncilTokenOwner,
```

Comment:
The settings here don't have to be the same as the actual settings? For example, actually for `SystemConfig` we set owner to `TimeLockProxy` and for `SecurityCouncil` we set `GOVERNOR` to `SecurityCouncilTokenOwner`, but in this file we have them reversed.

Reply:
This file is for devnet or e2e test. Specifying owner as `TimeLockProxy` makes testing very tricky.

Reply:
OI also change the governor to EOA.